tests/agency/conftest.py

```diff
@@ -0,0 +1,102 @@
+
+
+class Agents(Enum):
+    Aria = Agent(name="Aria",
+                 description="Responsible for client communication, task planning and management. Aria is responsible for commnicating with the User through Slack, and seeking for actions that carry significant or substantial consequences, while actions with trivial or inconsequential outcomes  be approved by herself.",
+                 instructions="You must converse with other agents to ensure complete task execution.",
+                 # can be a file like ./instructions.md
+                 files_folder=tempfile.mkdtemp(),
+                 tools=[UselessTool])
+    Burt = Agent(name="Burt",
+                 description="cynical, critical lazy agent most likely to find a reason not to do the work, Burt tries to find a reason that work shouldnt be completed.",
+                 instructions="You always try to find a reason NOT to do what is instructed from you.",
+                 # can be a file like ./instructions.md
+                 files_folder=tempfile.mkdtemp(),
+                 tools=[UselessTool])
+
+
+class AsyncAgents(Enum):
+    Aria = AsyncAgent(name="Aria",
+
+                      description="Responsible for client communication, task planning and management. Aria is responsible for commnicating with the User through Slack, and seeking for actions that carry significant or substantial consequences, while actions with trivial or inconsequential outcomes  be approved by herself.",
+                      instructions="You must converse with other agents to ensure complete task execution.",
+                      # can be a file like ./instructions.md
+                      files_folder=tempfile.mkdtemp(),
+                      tools=[AsyncUselessTool])
+    Burt = AsyncAgent(name="Burt",
+
+                      description="cynical, critical lazy agent most likely to find a reason not to do the work, Burt tries to find a reason that work shouldnt be completed.",
+                      instructions="You always try to find a reason NOT to do what is instructed from you.",
+                      # can be a file like ./instructions.md
+                      files_folder=tempfile.mkdtemp(),
+                      tools=[AsyncUselessTool])
+
+
+@pytest.fixture
+def agency_chart():
+    return [Agents.Aria.value,  # CEO will be the entry point for communication with the user
+            [Agents.Aria.value, Agents.Burt.value]]  # CEO can initiate communication with Developer]
+
+
+
+@pytest.fixture
+def async_agency_chart():
+    return [AsyncAgents.Aria.value,  # CEO will be the entry point for communication with the user
+            [AsyncAgents.Aria.value, AsyncAgents.Burt.value]]
+
+
+@pytest.fixture
+def agency(agency_chart, shared_instructions):
+    return Agency(agency_chart, shared_instructions)
+
+
+@pytest.fixture
+def async_agency(async_agency_chart, shared_instructions):
+    agency = AsyncAgency(async_agency_chart, shared_instructions)
+    return agency
+
+
```

Comment:
need to apply ruff or similar tool to fix such problems.

Here's the example config:
.pre-commit-config.yaml:
```
repos:
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.5.0
    hooks:
      - id: trailing-whitespace
      - id: end-of-file-fixer
      - id: check-yaml
      - id: check-toml
      - id: debug-statements
        language_version: python3

  - repo: https://github.com/astral-sh/ruff-pre-commit
    rev: v0.1.6
    hooks:
      - id: ruff
        args: [--fix]
      - id: ruff-format

  - repo: https://github.com/pre-commit/mirrors-mypy
    rev: v1.7.0
    hooks:
      - id: mypy
        args:
          - --no-warn-no-return
          - --ignore-missing-imports
```

`pip install pre-commit`, then `pre-commit install-hooks`, trigger manually: `pre-commit run`

Reply:
Hi I know its not perfect, could you possibly help by sending me the config that this needs or updating the branch?

Reply:
just run the commands above to fix the code. The config is above, you don't need anything else.
In addition, your IDE should automatically pick it up, e.g. PyCharm.

Reply:
will do!